RELEASE_NOTES.md

```diff
@@ -19,6 +19,34 @@ Notable changes include:
 
   * Bug fixes/improvements:
 
+Version 2022.10.3 -- Release date 2022-12-01
+============================================
+
+This release fixes a few issues that were found after the v2022.10.2 release.
+
+Notable changes include:
+
+  * Update camp submodule to v2022.10.1
+  * Update BLT submodule to commit 8c229991 (includes fixes for crayftn + hip)
+
```

Comment:
is this an extra newline?

Reply:
That was intentional to emphasize the submodule version changes.